tests/backends/base/test_schema.py

```diff
@@ -0,0 +1,22 @@
+from django.db import connection, models
+from django.test import TestCase
+
+
+class SchemaEditorTests(TestCase):
```

Comment:
You can probably use `SimpleTestCase` with `allow_database_queries = True` since this isn't performing any database level alteration. That should save you two 4 queries (`BEGIN; SAVEPOINT, ROLLBACK TO; ROLLBACK`).